ci/do_ci.sh

```diff
@@ -90,6 +97,33 @@ function do_tsan() {
     run_bazel test ${BAZEL_TEST_OPTIONS} -c dbg --config=clang-tsan //test/...
 }
 
+function cleanup_benchmark_artifacts {
+    # TODO(oschaaf): we clean the tmp dir above from uninteresting stuff
+    # that crept into the tmp/output directory. The cruft gets in there because
+    # other tooling also responds to the TMPDIR environment variable, which in retrospect
+    # was a bad choice.
+    # Consider using a different environment variable for the benchmark tooling
+    # to use for this.
+    rm -rf ${TMPDIR}/tmp.*
```

Comment:
This seems very scary of `${TMPDIR}` isn't set. It might be a good idea to verify that first.

Reply:
I safeguarded this for a bit by ensuring tmpdir is > 4 characters and the directory exists, wdyt?

Reply:
SGTM